lightning/src/offers/invoice.rs

```diff
@@ -705,6 +705,16 @@ macro_rules! invoice_accessors { ($self: ident, $contents: expr) => {
 		$contents.supported_quantity()
 	}
 
+	/// The public key used by the recipient to sign invoices.
+	///
+	/// From [`Offer::issuer_signing_pubkey`] and may be `None`; also `None` if the invoice was
+	/// created in response to a [`Refund`].
+	///
+	/// [`Offer::issuer_signing_pubkey`]: crate::offers::offer::Offer::issuer_signing_pubkey
```

Comment:
Maybe we can reword it a little bit to make it more clear. How about?

```suggestion
	  /// Returns the public key used by the recipient to sign invoices.
  /// The returned value varies based  on the context in which the invoice was created:
	/// If the invoice was created in response to an [`Offer`] then it returns the [`Offer::issuer_signing_pubkey`]
	/// If the invoice was created in response to a [`Refund`] then it returns `None`. 
	///
	/// [`Offer::issuer_signing_pubkey`]: crate::offers::offer::Offer::issuer_signing_pubkey
```

Reply:
This isn't correct. It may return `None` when created in response to an `Offer` as detailed in `Offer::issuer_signing_pubkey`. Rather than repeat those docs, I'd rather just link to them. Added some clarification in those docs for when it is `None`, though.